app/views/hyrax/collections/show.html.erb

```diff
@@ -11,7 +11,7 @@
 
 			<div class="hyc-title">
 				<h1><%=@presenter.title.first%></h1>
-				<span class="label label-success">Collection</span>
+				<span class="label label-success"><%= @presenter.collection_type.title %></span>
```

Comment:
The presenter has a method "collection_type_badge" which does the same thing and is used elsewhere (i.e. the dashboard show page, index page, etc). It doesn't actually make a badge, simply the text for the badge. I'd like to see all of these locations do it consistently.

Also, at least some of the other locations do not have the "badge" format, simply the text. If not part of this issue, perhaps we need another issue to make consistent formatting among all collection type "badge" locations?

Reply:
I just updated to use the `collection_type_badge` method in this file and re-pushed.   Maybe a new ticket is created to make these usage instances consistent in the application?